src/@utils/googleUrl.ts

```diff
@@ -0,0 +1,6 @@
+// check if the url is a google domain
+export const isGoogleUrl = (url: string): boolean => {
+  if (!url) return
+  const googleUrl = new URL(url)
+  return googleUrl.hostname.endsWith('google.com')
```

Comment:
This is intentional so that it covers all files hosted on any Google service e.g. `drive.google.com/abc` or `docs.google.com/123` etc

Reply:
yeah, I'm not sure how we can sanitize this url without having white listing.

Reply:
Not really sure what the point of adding that in here is? We are already checking if it is a valid url with `isUrl` in the [yup validation](https://github.com/oceanprotocol/market/blob/01df05218b042aea26df091c126dfde696d144d8/src/%40utils/yup.ts#L19) before we check if it is a google link

Reply:
Yeah, sorry. What I meant is that we can keep all url manipulations / checking under the same file: `/@utils/url/index` instead of creating a new one: `/@utils/googleUrl.ts` . At the end, we're checking an URL, so it makes sense to me to have them both in the same file.

Reply:
Yeah, makes sense. I've moved it into that file